machine-learning.ipynb

```diff
@@ -114,7 +119,7 @@
    "source": [
     "from sklearn.externals import joblib\n",
     "\n",
-    "with joblib.parallel_backend('dask', scatter=[X, y]):\n",
```

Comment:
@ogrisel what's the current state of auto-scattering? IIRC, there was a performance issue? I may be misremembering.